packages/date/src/DateRange.js

```diff
@@ -28,7 +28,7 @@ const AvDateRange = ({
   datepickerProps,
   'data-testid': dataTestId,
   datepicker,
-  autofill,
+  autosync,
```

Comment:
Should it be `autoSync` or `autosync`? `Autofill` made sense bc that is defined as a single word but not the other way? 📖 🤔

Reply:
🤓

Reply:
8a6220b6bedee839a0435fdf955fd9cf091fddab